pytest_robotframework/_internal/plugin.py

```diff
@@ -124,5 +137,6 @@ def pytest_runtest_setup(item: Item):
 def pytest_runtestloop(session: Session) -> object:
     if session.config.option.collectonly:  # type:ignore[no-any-expr]
         return None
+    keywordify(pytest, ["fail", "xfail", "raises", "deprecated_call", "warns"])
```

Comment:
keywordify `BuiltIn` and other robot modules

Reply:
also investigate whether the `xfail` function should be keywordified

Reply:
> keywordify `BuiltIn` and other robot modules

that causes double keywording in robot tests so im leaving it as is for now

> also investigate whether the `xfail` function should be keywordified

yes